sonic_y_cable/y_cable.py

```diff
@@ -28,6 +31,30 @@
 Y_CABLE_MUX_DIRECTION = 644
 Y_CABLE_ACTIVE_TOR_INDICATOR = 645
 Y_CABLE_MANUAL_SWITCH_COUNT = 669
+Y_CABLE_CONFIGURE_PRBS_TYPE = 768
+Y_CABLE_ENABLE_PRBS = 769
+Y_CABLE_INITIATE_BER_MEASUREMENT = 770
+Y_CABLE_TARGET = 794
+Y_CABLE_ENABLE_LOOPBACK = 793
+Y_CABLE_LANE_1_BER_RESULT = 771
+Y_CABLE_MAX_LANES = 2
+Y_CABLE_INITIATE_EYE_MEASUREMENT = 784
+Y_CABLE_LANE_1_EYE_RESULT = 785
+Y_CABLE_PN_NUMBER = 168
+Y_CABLE_VENDOR_NAME = 148
+Y_CABLE_MANUAL_SWITCH_COUNT = 653
+Y_CABLE_AUTO_SWITCH_COUNT = 657
+Y_CABLE_NIC_CURSOR_VALUES = 661
+Y_CABLE_TOR1_CURSOR_VALUES = 681
+Y_CABLE_TOR2_CURSOR_VALUES = 701
+Y_CABLE_NIC_LANE_ACTIVE = 721
+Y_CABLE_TARGET_NIC = 0
+Y_CABLE_TARGET_TOR1 = 1
+Y_CABLE_TARGET_TOR2 = 2
+Y_CABLE_EYE_PRBS_TARGET_LOCAL = 0
+Y_CABLE_EYE_PRBS_TARGET_TOR1 = 1
+Y_CABLE_EYE_PRBS_TARGET_TOR2 = 2
+Y_CABLE_EYE_PRBS_TARGET_NIC = 3
```

Comment:
Suggest separating TARGET and EYE_PRBS_TARGET from the other constants with blank lines, since they are conceptually different:

```
...
Y_CABLE_NIC_LANE_ACTIVE = 721

Y_CABLE_TARGET_NIC = 0
Y_CABLE_TARGET_TOR1 = 1
Y_CABLE_TARGET_TOR2 = 2

Y_CABLE_EYE_PRBS_TARGET_LOCAL = 0
Y_CABLE_EYE_PRBS_TARGET_TOR1 = 1
Y_CABLE_EYE_PRBS_TARGET_TOR2 = 2
Y_CABLE_EYE_PRBS_TARGET_NIC = 3
```

And maybe also add comments above each group to describe them

Also, unless you have a good reason not to, consider removing the `Y_CABLE_` prefix from all constants. When this module is imported, they should all be part of the `y_cable` namespace (assuming the calling application does not `import *`, which is bad practice and will generate LGTM alerts, so we shouldn't need to worry about it). Doing this will shorten all these constant names. On top of this, I would also suggest adding a `OFFSET_` prefix to all of the constants which represent byte offsets.

Reply:
fixed